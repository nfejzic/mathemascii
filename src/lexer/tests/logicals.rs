use super::Snapshot;

super::test_snap!(and_or_not, "andornot and or not");
super::test_snap!(implications, "implies <=> => iff");
super::test_snap!(quantors, "AA EE forallexists AAEE");
super::test_snap!(other, "_|_ bottop TT |-- |==");
