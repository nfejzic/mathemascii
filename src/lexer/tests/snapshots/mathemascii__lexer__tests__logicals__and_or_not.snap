---
source: src/lexer/tests/logicals.rs
expression: "Snapshot((input, tokens))"
---
andornot and or not

and
^^^ -> Logical(And) at: 0 -> 3

   or
   ^^ -> Logical(Or) at: 3 -> 5

     not
     ^^^ -> Logical(Not) at: 5 -> 8

         and
         ^^^ -> Logical(And) at: 9 -> 12

             or
             ^^ -> Logical(Or) at: 13 -> 15

                not
                ^^^ -> Logical(Not) at: 16 -> 19


